src/components/LiquidityChartRangeInput/index.tsx

```diff
@@ -151,9 +152,9 @@ export default function LiquidityChartRangeInput({
 
       const percent = (x < price ? -1 : 1) * ((Math.max(x, price) - Math.min(x, price)) / price) * 100
 
-      return price ? `${format(Math.abs(percent) > 1 ? '.2~s' : '.2~f')(percent)}%` : ''
+      return price ? `${(Math.sign(percent) < 0 ? '-' : '') + formatPercent(percent)}` : ''
```

Comment:
What was the previous logic here?

Reply:
it switched to two fixed two decimal points when the percent was between -1-0-1, and two sig figs when it was greater than 1. I thought it was just best to keep it to how we currently handle percentages